plugins/ch341a/fu-ch341a-cfi-device.c

```diff
@@ -106,14 +106,17 @@ fu_ch341a_cfi_device_read_jedec(FuCh341aCfiDevice *self, GError **error)
 		g_set_error_literal(error,
 				    FWUPD_ERROR,
 				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "flash ID non-valid");
+				    "flash ID non-valid, got 0x000000");
```

Comment:
maybe flash-ID is empty?

Reply:
It's really for the case where you forgot to connect a wire, e.g. CS